Localization Support

IBRS-4Ever opened the pull request:
Updated to version 3.4

IBRS-4Ever committed:
Localization Support

Updated to version 3.4

Blueberryy committed:
russian translation

IBRS-4Ever committed:
Merge pull request #1 from Blueberryy/master

russian translation

IBRS-4Ever committed:
Add files via upload

IBRS-4Ever committed:
Polish Localization

Stannowsky commented:
[enhanced_playermodel_selector.txt](https://github.com/Fesiug/Enhanced-PlayerModel-Selector/files/12321229/enhanced_playermodel_selector.txt)
Updated Polish localization files.
(rename .txt to .properties)

IBRS-4Ever committed:
Update enhanced_playermodel_selector.properties

SheepYhangCN committed:
Added Traditional Chinese

IBRS-4Ever committed:
Merge pull request #2 from SheepYhangCN/master

Added Traditional Chinese | 添加繁體中文

IBRS-4Ever committed:
3.7 Added TChinese

NovaDiablox committed:
Added Turkish Language

IBRS-4Ever committed:
Merge pull request #3 from NovaDiablox/patch-1

Added Turkish Language

IBRS-4Ever committed:
Tr

Fesiug commented:
I'll push this soon!!

Fesiug commented:
I pushed some tweaks that change a few lines. Check up and tweak whatever you must!

IBRS-4Ever committed:
Update lf_playermodel_selector.lua

IBRS-4Ever committed:
Next Anim Update

GermanMako committed:
Create enhanced_playermodel_selector.properties

IBRS-4Ever committed:
Merge pull request #4 from GermanMako/GermanMaKo

Added German Language

IBRS-4Ever committed:
DE

IBRS-4Ever committed:
Merge branch 'master' of https://github.com/IBRS-4Ever/Enhanced-PlayerModel-Selector

IBRS-4Ever committed:
Hide Default PMs

IBRS-4Ever committed:
Localization

Stannowsky commented:
[enhanced_playermodel_selector.txt](https://github.com/user-attachments/files/16715227/enhanced_playermodel_selector.txt)
Updated localization files.

IBRS-4Ever committed:
Polish Translation

IBRS-4Ever committed:
Fesiug & dar-su's Updates

IBRS-4Ever committed:
More Updates

IBRS-4Ever committed:
Server fix?

Stannowsky commented:
[enhanced_playermodel_selector.txt](https://github.com/user-attachments/files/16857176/enhanced_playermodel_selector.txt)
I re-wrote most of the Polish translation, due to my poor skills at the time :p
(altough not that much changed really, punctuation corrections, big and smol letter fixes, etc.)

IBRS-4Ever committed:
Pl update

NovaDiablox committed:
Updated Turkish L10n

Added/translated missing lines
Couple of fixes

NovaDiablox committed:
update

NovaDiablox committed:
another one

NovaDiablox committed:
...

IBRS-4Ever committed:
Merge pull request #6 from NovaDiablox/patch-1

Updated Turkish L10n

IBRS-4Ever committed:
HTTPS update.

IBRS-4Ever committed:
DPanel Update

IBRS-4Ever committed:
Fixed Hands "Has Bodygroups" check

NovaDiablox committed:
Added Czech (Machine Translation)

NovaDiablox committed:
Added Korean (Machine Translation)

IBRS-4Ever committed:
Merge pull request #7 from NovaDiablox/patch-1

Added Czech & Korean (Machine Translation)

IBRS-4Ever committed:
Update lf_playermodel_selector.lua

IBRS-4Ever committed:
Update lf_playermodel_selector.lua

Commit:
Create enhanced_playermodel_selector.properties

Commit:
fix

iosis9 committed:
fix2

IBRS-4Ever committed:
Merge pull request #8 from iosis9/master

Spanish Translation

IBRS-4Ever committed:
Update lf_playermodel_selector.lua

IBRS-4Ever committed:
Localize Bodygroups.

NovaDiablox committed:
Updated Turkish L10n (Body Groups)

IBRS-4Ever committed:
Merge pull request #9 from NovaDiablox/bgl10n

Updated Turkish L10n (Body Groups)

IBRS-4Ever committed:
Bodygroup localization file.

NovaDiablox committed:
tr

NovaDiablox committed:
zh cn

NovaDiablox committed:
en

NovaDiablox committed:
Fixes

NovaDiablox committed:
update en

NovaDiablox committed:
update tr

NovaDiablox committed:
Update en

IBRS-4Ever committed:
Merge pull request #10 from NovaDiablox/bgl10n

Add New Body Group Strings

IBRS-4Ever committed:
rename

IBRS-4Ever committed:
Update lf_playermodel_selector.lua

IBRS-4Ever committed:
=

Satton2 committed:
Localization (Russian and English) update + small visual tweaks

- Updated Russian localization:
- - Updated existing strings (fixed some issues / mistakes, rephrased some parts).
- - Localized new strings.
- Updated English localization:
- - Changed "playermodel" to "player model"
- - Updated some strings (removed some unnecessary shortenings, added some needed spaces, changed some casing, rephrased some sentences).
- Small visual tweaks:
- - Resized some button sizes and positions to make it look less "skinny" and to fill some unnecessary free spaces.

Satton2 committed:
Russian localization update

From PR: https://github.com/IBRS-4Ever/Enhanced-PlayerModel-Selector/pull/10

IBRS-4Ever committed:
Merge pull request #11 from Satton2/master

Localization (Russian and English) update + small visual tweaks

IBRS-4Ever committed:
Update enhanced_playermodel_selector_bodygroups.properties

IBRS-4Ever committed:
Update lf_playermodel_selector.lua

IBRS-4Ever committed:
Update lf_playermodel_selector.lua

NovaDiablox committed:
Added New Strings

NovaDiablox committed:
Updated Turkish L10n

NovaDiablox committed:
Add More BG Strings

NovaDiablox committed:
Update Turkish

NovaDiablox committed:
German Bodygroup Localization By @GermanMako

German Bodygroup Localization By @GermanMako
https://github.com/GermanMako

IBRS-4Ever committed:
Update lf_playermodel_selector.lua

IBRS-4Ever committed:
Fixed Errors.

NovaDiablox committed:
Update README.md

Added credits.

PlucktheDragon commented:
I'm making a french/french canadian localization for the addon.

PlucktheDragon commented:
[enhanced_playermodel_selector_fr_fr-CA.txt](https://github.com/user-attachments/files/23563885/enhanced_playermodel_selector_fr_fr-CA.txt)
Here you go. I made some fixes to the french translation.

IBRS-4Ever committed:
Added FR Localization.

IBRS-4Ever committed:
Updated Credits.

PlucktheDragon commented:
[enhanced_playermodel_selector_bodygroups.properties (french, french-canadian).txt](https://github.com/user-attachments/files/24546884/enhanced_playermodel_selector_bodygroups.properties.french.french-canadian.txt)

I started and finished the french translation for the body groups properties.
